Eatery/Controllers/EateryTabBarController.swift

```diff
@@ -30,22 +31,32 @@ class EateryTabBarController: UITabBarController {
         brbNavigationController.navigationBar.barStyle = .black
         brbNavigationController.tabBarItem = UITabBarItem(title: nil, image: UIImage(named: "infoIcon.png"), tag: 2)
 
-        let navigationControllers = [eateryNavigationController, lookAheadNavigationController, brbNavigationController]
+        let navigationControllers = [eateriesNavigationController, lookAheadNavigationController, brbNavigationController]
         navigationControllers.forEach { $0.tabBarItem.imageInsets = UIEdgeInsets(top: 6, left: 0, bottom: -6, right: 0) }
+
         setViewControllers(navigationControllers, animated: false)
     }
-
+    
+    func tabBarControllerSupportedInterfaceOrientations(_ tabBarController: UITabBarController) -> UIInterfaceOrientationMask {
+        return .portrait
+    }
+    
+    func tabBarControllerPreferredInterfaceOrientationForPresentation(_ tabBarController: UITabBarController) -> UIInterfaceOrientation {
+        return .portrait
+    }
 
 }
 
 extension EateryTabBarController: UITabBarControllerDelegate {
 
-    func tabBarController(_ tabBarController: UITabBarController, didSelect viewController: UIViewController) {
-        if let navVC = viewController as? UINavigationController, let eateriesVC = navVC.viewControllers.first as? EateriesViewController {
-            eateriesVC.scrollToTop()
-        } else if let navVC = viewController as? UINavigationController, let lookAheadVC = navVC.viewControllers.first as? LookAheadViewController {
-            lookAheadVC.scrollToTop()
+    func tabBarController(_ tabBarController: UITabBarController, shouldSelect viewController: UIViewController) -> Bool {
```

Comment:
Why does this function now return a bool if the older version did not and still, presumably, worked?

Reply:
https://developer.apple.com/documentation/uikit/uitabbarcontrollerdelegate/1621166-tabbarcontroller

And also the old method did not work.